stories/victory-histogram.js

```diff
@@ -0,0 +1,955 @@
+/*eslint-disable no-magic-numbers*/
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { VictoryHistogram } from "../packages/victory-histogram/src";
+import { VictoryLine } from "../packages/victory-line/src";
+import { VictoryScatter } from "../packages/victory-scatter/src";
+import { VictoryTooltip } from "../packages/victory-tooltip/src";
+import { VictoryStack } from "../packages/victory-stack/src";
+import { VictoryTheme } from "../packages/victory-core/src";
+import { getChartDecorator } from "./decorators";
+import { getData } from "./data";
+
+import * as d3Array from "d3-array";
+import * as d3Scale from "d3-scale";
+import * as d3Time from "d3-time";
+
+const data = [
+  { x: 18 },
+  { x: 85 },
+  { x: 27 },
+  { x: 62 },
+  { x: 26 },
+  { x: 90 },
+  { x: 85 },
+  { x: 60 },
+  { x: 85 },
+  { x: 21 },
+  { x: 86 },
+  { x: 89 },
+  { x: 60 },
+  { x: 82 },
+  { x: 70 },
+  { x: 22 },
+  { x: 68 },
+  { x: 79 },
+  { x: 18 },
+  { x: 76 },
+  { x: 32 },
+  { x: 26 },
+  { x: 18 },
+  { x: 63 },
+  { x: 71 },
+  { x: 98 },
+  { x: 91 },
+  { x: 22 },
+  { x: 25 },
+  { x: 91 },
+  { x: 18 },
+  { x: 49 },
+  { x: 18 },
+  { x: 79 },
+  { x: 56 },
+  { x: 18 },
+  { x: 18 },
+  { x: 28 },
+  { x: 79 },
+  { x: 44 }
+];
+
+const timeData = [
+  { x: new Date("2018-09-24T07:00:00.000Z") },
```

Comment:
Can you move this to a new file to make the stories a bit easier to read through?